### ISSUE TITLE
Do you support mock location?

### ISSUE THREAD

**Issue by fre391** [Lyokone/flutterlocation#416]
Hi,

I would like to know, if you can also detect mock location?
Does it support Android and IOS?

Thanks

Markus

**Comment by GadepalliAditya1998** [comment]
currently this plugin does't support mock location, but for timebeing you can add a method to it. as it uses fused location API so it has a method to detect whether it is from mock location or not. 

Note: It only supports device >= Lollipop

**Comment by iampopal** [comment]
I am also looking for the isMocked location 

it will be great to add isMocked bool to LocationData

**Comment by xpat** [comment]
I came across this problem just the other day while completing module 13 Clima of Angela Yu's "The Complete Flutter Development Bootcamp Using Dart" (a Google recommended [codelab](https://flutter.dev/docs/codelabs).  She explains how the emulators in iOS and Android can be used to get a "custom location" (aka mock location?).  I didn't pay much attention to that part as I'm only using Flutter for web development.  Anyone know how to use the Chrome browser to get mock locations?  I probably do need this feature in the Location package.  (BTW, I used the Location, LocationData pub dev packages as a replacement for the Baseflow's Geolocator package).  The exact part of the lesson on custom locations starts around minute 7:56, in the "Refactoring the Location Method" under Section 13 (Clima) of the Bootcamp class.
